src/plot/series/label-series.js

```diff
@@ -26,6 +26,12 @@ import {ANIMATED_SERIES_PROPS} from 'utils/series-utils';
 const predefinedClassName = 'rv-xy-plot__series rv-xy-plot__series--label';
 
 class LabelSeries extends AbstractSeries {
+  getTextAnchor = (labelAnchorX, leftOfMiddle) => {
```

Comment:
It'd be a little more in the style of the project to make these defined outside of LabelSeries (ie just above) in order to avoid this's.